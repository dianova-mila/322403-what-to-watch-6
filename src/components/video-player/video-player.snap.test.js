import React from 'react';
import {render} from '@testing-library/react';
import VideoPlayer from "./video-player";

const src = `path`;

test(`VideoPlayer should render correctly`, () => {
  const {container} = render(<VideoPlayer src={src}/>);
  expect(container).toMatchSnapshot();
});
