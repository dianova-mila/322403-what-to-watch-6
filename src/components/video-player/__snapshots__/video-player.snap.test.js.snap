// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPlayer should render correctly 1`] = `
<div>
  <video
    autoplay=""
    height="175"
    src="path"
    width="280"
  />
</div>
`;
